apps/condo/pages/ticket/index.tsx

```diff
@@ -619,27 +625,13 @@ const FiltersContainer = ({ filterMetas }) => {
         changeSearch(e.target.value)
     }, [changeSearch])
 
-    let checkboxColSpan = 24
-    let filterButtonColSpan = 24
-
     const isXlContainerSize = contentWidth >= 980
-    const isXxlContainerSize = contentWidth >= 1288
-
-    if (isXlContainerSize) {
-        checkboxColSpan = 17
-        filterButtonColSpan = 7
-    }
-
-    if (isXxlContainerSize) {
-        checkboxColSpan = 18
-        filterButtonColSpan = 6
-    }
 
     return (
         <>
             <TableFiltersContainer ref={setRef}>
                 <Row gutter={FILTERS_CONTAINER_ROW_GUTTER} align='middle'>
-                    <Col span={24}>
+                    <Col span={isXlContainerSize ? 3 : 24}>
```

Comment:
Isn't that too little? I think it would be inconvenient to enter, for example, the address